src/change_stream.ts

```diff
@@ -923,7 +927,10 @@ export class ChangeStream<
     }
 
     if (isResumableError(changeStreamError, this.cursor.maxWireVersion)) {
-      this.cursor.close();
+      this.cursor.close().then(
```

Comment:
can we `.catch` or go with the rule disable approach here? I'm all for being explicit, but the `.then` tends to suggest that there might be a result returned from this function that one *could* care about, and that's not true for functions that don't return anything

Reply:
went with .catch(() => null) (still returning a value but null is the absence of something 🤔)